### ISSUE TITLE
BUG: Still issue with calendars displayed in quick view

### ISSUE THREAD

**Issue by noticz** [milowg/GMinder#14]
This is how it went down...

- Had the problem before, uninstalled and reinstalled everything seemed to fix the problem.
- Used it for about a week but only used one of the calendars "Work".
- Then I added something to "Mobile" calendar and it lost the "Work" calendar (just not in quick add drop down menu).
- Since it lost the "Work" calendar I needed to add it anyways so I added it to the "Work Schedule" calendar and then it lost my "Mobile" calendar.
- So to sum it up, all I have in my drop down list now under the quick add is the "Work Schedule" calendar.

Not sure if this is just a problem I am having or what but it seems that when it is looping through the calendars that it is coming to a bug and then just is not listing what comes after the bug. If you need me to send you anything to take a look at please let me know.

**Comment by noticz** [comment]
I got a look at the code and I found the bug that I am referring to...

In Create.cs there is a break that breaks the loop in the adding calendars to the drop down, it shouldn't have the break there at all since there is only one possible calendar in the array that can match what is in the settings file.
```
foreach (var cal in Schedule.Current)
{
	calendarList.Items.Add(cal);
	if (cal.Name == Properties.Settings.Default.LastQuickAdd)
	{
		calendarList.SelectedItem = cal;
		break;
	}
}
```

**Comment by milowg** [comment]
Fixed in the latest version (now available for download). Thanks for the help @noticz!

**Comment by noticz** [comment]
Looks like you fixed it in the code but I think the install file online was not updated correctly. Download saves it as GMinder-1.4.0.3.exe but still the same problem and program still says 1.4.0.2 under About.

**Comment by milowg** [comment]
Sorry about that. I've updated the install and it should be the correct version now.
